pkg/cmd/server/start/master_args.go

```diff
@@ -180,8 +182,11 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 			ClientCert: configapi.CertInfo{},
 		},
 
-		MasterAuthorizationNamespace:      "master",
-		OpenShiftSharedResourcesNamespace: "openshift",
+		PolicyConfig: configapi.PolicyConfig{
+			BootstrapPolicyFile:               args.PolicyArgs.PolicyFile,
+			MasterAuthorizationNamespace:      "master",
```

Comment:
make `master` and `openshift` constants and reference here and in `CreateBootstrapPolicy`